src/pubtools/_marketplacesvm/tasks/push/command.py

```diff
@@ -215,6 +215,7 @@ def _push_upload(
                 pi,
                 custom_tags=mapped_item.get_tags_for_marketplace(marketplace),
                 accounts=meta.get("sharing_accounts", []),
+                snapshot_accounts=meta.get("snapshot_accounts", []),
```

Comment:
@JAVGan I though we need tit for community workflow and not for stratosphere
and looks like we already have it in the community workflow
* https://github.com/release-engineering/pubtools-marketplacesvm/blob/1009389c185ca84ac926242f8b30da838ec52d2c/src/pubtools/_marketplacesvm/tasks/community_push/command.py#L501
* https://github.com/release-engineering/pubtools-marketplacesvm/blob/1009389c185ca84ac926242f8b30da838ec52d2c/src/pubtools/_marketplacesvm/tasks/community_push/command.py#L388
* https://github.com/release-engineering/pubtools-marketplacesvm/blob/1009389c185ca84ac926242f8b30da838ec52d2c/src/pubtools/_marketplacesvm/tasks/community_push/command.py#L285

But related slack discussion is still in my TODO list.

Reply:
Great! I thought we would need it for the `stratosphere` workflow as well... Do you still think it makes sense to include it on such workflow? Or is it better to close the PR?